dashboard/app/models/lesson.rb

```diff
@@ -63,8 +63,10 @@ class Lesson < ApplicationRecord
   )
 
   # A lesson has an absolute position and a relative position. The difference between the two is that relative_position
-  # only accounts for other lessons that have the same lockable setting, so if we have two lockable lessons followed
-  # by a non-lockable lesson, the third lesson will have an absolute_position of 3 but a relative_position of 1
+  # numbers the lessons in order in two groups 1. lessons that are numbered (lockable false OR has_lesson_plan true)
+  # 2. lessons that are not numbered (lockable true AND has_lesson_plan false)
+  # if we have two lessons without lesson plans that are lockable followed by a
+  # lesson that is not lockable, the third lesson will have an absolute_position of 3 but a relative_position of 1
```

Comment:
numbered and unnumbered sounds great! I would just add in the comments here that "numbered" means that the display name is prefixed with the lesson number, or some other wording -- otherwise it was a bit confusing because I thought it was referring to urls at first, where "unnumbered" lessons are still numbered.